gnucash/gtkbuilder/dialog-sx.glade

```diff
@@ -1231,7 +1231,7 @@
                             </child>
                             <child>
                               <object class="GtkRadioButton" id="rb_num_occur">
-                                <property name="label" translatable="yes">For</property>
+                                <property name="label" translatable="yes">For Time(s)</property>
```

Comment:
Same problem as with the preferences dialog above, plus 'Time(s)' isn't as clear as 'occurrences'.

That aside in both cases these are effectively composed strings, the equivalent of writing `_("For") << secs << _("occurrences")` instead of `_("For %d occurrences")`. It may be impossible to translate correctly into some languages.

Reply:
> instead of `_("For %d occurrences")`

Because %d is the input, the rest output.
But
```
comments="Begin of 'For %d occurrences'"
…
comments="End of 'For %d occurrences'"
```
could help.

Reply:
Better but still won't help for languages where the string would be "%d for occurrences" or "for occurrences %d".